### PR TITLE
RSDK-2380 Check if labels file is split by spaces or commas

### PR DESCRIPTION
**Pull request by kim-mishra**

### CODE REVIEW

File: services/vision/builtin/tflite_classifier.go [hunk 1]
```diff
@@ -82,9 +83,9 @@ func NewTFLiteClassifier(ctx context.Context, conf *vision.VisModelConfig,
 		if err != nil {
 			return nil, err
 		}
-
 		classifications, err := unpackClassificationTensor(ctx, outTensor, model, labels)
 		if err != nil {
+			logger.Error(err)
```
Comment (kim-mishra) [inline services/vision/builtin/tflite_classifier.go:88]: What is the best way to surface this error to the user? Logging it here logs it over and over and over again, but just returning the error doesn't seem to surface it

Comment (kharijarrett) [reply services/vision/builtin/tflite_classifier.go:88]: Yeah this function that gets returned will run every time an image gets classified. My first thought is that if we hate that, maybe try a logger.Fatal() which will kill everything after printing it out once. If that seems like overkill, maybe only fatal depending on the type of error. Either way it seems like don't want to keep on trying to classify this over and over again if we're definitely not unpacking the tensors correctly.

Comment (kim-mishra) [reply services/vision/builtin/tflite_classifier.go]: hmm the linter complains about `logger.Fatal`, do you know of a workaround?

File: services/vision/builtin/tflite_test.go [hunk 2]
```diff
@@ -254,3 +255,23 @@ func TestMoreClassifierModels(t *testing.T) {
 	test.That(t, bestClass[0].Label(), test.ShouldResemble, "292")
 	test.That(t, bestClass[0].Score(), test.ShouldBeGreaterThan, 0.93)
 }
+
+func TestInvalidLabels(t *testing.T) {
```
Comment (bhaney) [inline services/vision/builtin/tflite_test.go:259]: Could you also add a test of it successfully unpacking a label file where all the labels are on one string, comma separated?

**Review by bhaney — CHANGES_REQUESTED** [review]
Nice! Just one rearranging of lines, and one test to add

**Review by bhaney — APPROVED** [review]
LGTM

**Review by kharijarrett — APPROVED** [review]
Nice LGTM!

**Pull request merged**